docs/man_pages/lib-management/plugin-add.md

```diff
@@ -24,15 +24,17 @@ Add the default version of an Apache Cordova plugin and set all variables from t
 Usage | Synopsis
 ------|---------
 List custom npm or NativeScript modules | `$ appbuilder plugin add --available [--count]`
-Add custom npm or NativeScript modules | `$ appbuilder plugin add <Name or ID>`
-Add a specific version of a custom npm or NativeScript module | `$ appbuilder plugin add <Name or ID>@<Version>`
-Add a custom npm or NativeScript module from GitHub URL | `$ appbuilder plugin add <URL>`
-Add a custom npm or NativeScript module from local path | `$ appbuilder plugin add <Path>`
-Add custom npm or NativeScript module and set all variables from the command line | `$ appbuilder plugin add <Name or ID or URL or Path> --var.<Variable ID> <Variable Value>`[\*\*](#note)
-Add a specific version of a custom npm or NativeScript module and set all variables from the command line | `$ appbuilder plugin add <Name or ID>@<Version> --var.<Variable ID> <Variable Value>`[\*\*](#note)
+Add custom npm or NativeScript modules | `$ appbuilder plugin add <Name or ID> [--no-types]`
+Add a specific version of a custom npm or NativeScript module | `$ appbuilder plugin add <Name or ID>@<Version> [--no-types]`
+Add a custom npm or NativeScript module from GitHub URL | `$ appbuilder plugin add <URL> [--no-types]`
+Add a custom npm or NativeScript module from local path | `$ appbuilder plugin add <Path> [--no-types]`
+Add custom npm or NativeScript module and set all variables from the command line | `$ appbuilder plugin add <Name or ID or URL or Path> --var.<Variable ID> <Variable Value> [--no-types]`[\*\*](#note)
+Add a specific version of a custom npm or NativeScript module and set all variables from the command line | `$ appbuilder plugin add <Name or ID>@<Version> --var.<Variable ID> <Variable Value> [--no-types]`[\*\*](#note)
 
 <% if(isHtml) { %><a id="note"></a><% } %>
 \*\* If the NativeScript plugin has multiple variables, you can set `--var` for each variable.
+
+For TypeScript projects the command will try to install `.d.ts` files for the specified module from `@types`. In case you do not want to install the `.d.ts` files, you can pass `--no-types` option.
```

Comment:
For TypeScript projects, the command will try to install `.d.ts` files for the specified module from `@types`. If you do not want to install the `.d.ts` files, you can set the `--no-types` flag.